nautobot/dcim/tests/test_graphql.py

```diff
@@ -83,3 +83,55 @@ def test_execute_query(self):
             self.assertEqual(
                 resp["data"]["devices"][0]["platform"]["network_driver_mappings"]["scrapli"], "cisco_iosxe"
             )
+
+        with self.subTest("device serial number query"):
+            non_empty_serial_device = Device.objects.first()
+            non_empty_serial_device.serial = "1234567890"
```

Comment:
Can we use a serial that contains letters so that we can assert that the default filtering is still case-insensitive?